dts/bindings/power-domain/intel,power-domain.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Power domains for intel ADSP
+
+compatible: "intel,power-domain"
+
+include: power-domain.yaml
+
+properties:
+    write_address:
+      type: int
+      required: true
+      description: Address to set PD state
+
+    read_address:
+      type: int
+      required: false
+      description: Address to read PD state
+
+    bit:
```

Comment:
I think this property should be more meaningful or, at least, have a better description. Read / write address is kind of self explanatory, but "bit" ??? what does it mean in this context ?

Reply:
Changed to bit_position and added a more detailed description.